mlflow/langchain/__init__.py

```diff
@@ -293,11 +299,10 @@ def log_model(
     import langchain
 
     if not isinstance(
-        lc_model, (langchain.chains.llm.LLMChain, langchain.agents.agent.AgentExecutor)
+        lc_model, (langchain.chains.Chain, langchain.agents.agent.AgentExecutor)
     ):
         raise mlflow.MlflowException.invalid_parameter_value(
-            "MLflow langchain flavor only supports logging langchain.chains.llm.LLMChain and "
-            + f"langchain.agents.agent.AgentExecutor instances, found {type(lc_model)}"
+            _UNSUPPORTED_MODEL_ERROR_MESSAGE.format(instance_type=str(type(lc_model)))
```

Comment:
```suggestion
            _UNSUPPORTED_MODEL_ERROR_MESSAGE.format(instance_type=type(lc_model).__name__)
```

Do we need `__name__` here?

```python
>>> str(type(0))
"<class 'int'>"
>>> str(type(0).__name__)
'int'
```